repo-scripts/repo-toolbox/src/BumpCyclicsAction.ts

```diff
@@ -84,7 +84,7 @@ export class BumpCyclicsAction extends CommandLineAction {
       const childProcess: ChildProcess = Executable.spawn('npm', ['view', packageName, 'version']);
```

Comment:
Make sure we set `stdio` to `['ignore', 'pipe', 'pipe']` here

Reply:
Fixed